packages/turf-kinks/index.js

```diff
@@ -1,8 +1,8 @@
 /**
- * Takes a {@link Polygon|polygon} and returns {@link Point|points} at all self-intersections.
+ * Takes a {@link LineString|linestring}, {@link MultiLineString|multi-linestring}, {@link MultiPolygon|multi-polygon}, or {@link Polygon|polygon} and returns {@link Point|points} at all self-intersections.
  *
  * @name kinks
- * @param {Feature<Polygon>|Polygon} polygon input polygon
+ * @param {Feature<LineString>|Feature<MultiLineString>|Feature<MultiPolygon>|Feature<Polygon>|LineString|MultiLineString|MultiPolygon|Polygon} feature input feature
```

Comment:
Might want to make the JSDocs simpler.
`* @param {Feature<LineString|MultiLineString|MultiPolygon|Polygon>} feature`

The complex typings can be defined in the Typescript definition. https://github.com/Turfjs/turf/blob/master/packages/turf-kinks/index.d.ts